docs/software/camera.md

```diff
@@ -6,16 +6,17 @@
 - [**MotionEye - Web interface & surveillance for your camera**](#motioneye)
 - [**mjpg-streamer - Simple camera streaming tool with HTML plugin**](#mjpg-streamer)
 
-??? info "How do I run **DietPi-Software** and install **optimised software**?"
-    To install any of the **DietPi optimised software** listed below run from the command line:
+??? info "How do I run **DietPi-Software** and install **optimised software** items?"
+    To install any of the **DietPi optimised software items** listed below run from the command line:
 
     ```sh
     dietpi-software
     ```
 
-    Choose **Software Optimised** and select one or more items. Finally click on `Install`. DietPi will do all the necessary steps to install and start these software items.
+    Choose **Browse Software** and select one or more items. Finally select `Install`.  
+    DietPi will do all the necessary steps to install and start these software items.
 
-    ![DietPi-Software menu screenshot](../assets/images/dietpi-software.jpg){: width="643" height="365" loading="lazy"}
```

Comment:
Let's keep `DietPi-Software menu screenshot` here and in all other cases of that image. I intentionally changed it (in all cases) so that the image alt tag better serves it's actual purpose to explain to users what they are expected to see, if the image is not loaded and especially for visually impaired users which use screen readers. "DietPi software" says nearly nothing about the content in the context this appears, so then it would be better to leave it empty `[](../assets/images/dietpi-software.jpg)` 😉.

Reply:
Ok, changed it at all places, no it is consistent over all documents.